ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -1140,7 +1169,10 @@ - (void)flipToNextPageFrom:(ORKStepViewController *)fromController {
         return;
     }
     
-    ORKStep *step = [self nextStep];
+    ORKStep *step = fromController.parentReviewStep;
+    if (!step) {
+        step = [self nextStep];
+    }
```

Comment:
Since the animation has been changed, shall we replace this code with error logging?